adding tsc blog post (#1534)
withastro/astro.build: src/content/blog/astro-tsc-2025.mdx

---
title: '2025 Technical Steering Committee'
description: 'Meet Astro’s 2025 Technical Steering Committee and learn how our open governance model supports the framework’s growth, performance, and community-led development.'
publishDate: '2025-04-18'
authors:
  - fred
coverImage: '/src/content/blog/_images/astro-agency-partners/agency-program.webp'
socialImage: '/src/content/blog/_images/astro-agency-partners/og-agency-program.webp'
lang: 'en'
---

> The Astro project has a new Technical Steering Committee (TSC) for 2025! I am excited to announce **Emanuele Stoppa**, **Florian Lefebvre,** and **Matt Kane** are now part of the Astro TSC, joining Matthew Phillips who will remain on the TSC. Additionally, we are proud to recognize Matt Kane as the official Astro framework team lead going forward, joining Sarah Rainsberger who will continue as the project's official docs team lead.

Author: totally forgot about this! will use it

The TSC was formed to be ultimately responsible for the long-term health and growth of the Astro codebase. To quote directly from the [official Astro governance](https://github.com/withastro/.github/blob/main/GOVERNANCE.md):

> *A TSC member guides the direction of the project and ensures a healthy future for the Astro codebase. TSC members are ultimately responsible for technical decision making when it comes to any changes to the Astro codebase.*
> 
> *A TSC member has significant sway in software design decisions. For this reason, coding experience is critical for this role. TSC membership is one of the only roles that requires a significant contribution history of code to the Astro project on GitHub.*
>
> *The TSC is limited to a maximum of **5** members at any one time. This size ensures adequate coverage of important areas of expertise balanced with the ability to make decisions efficiently.*
>

> > *The Steward is not eligible for the TSC, but may participate in TSC discussions and has a role in guiding the conversation to consensus (see [<u>Voting: RFC Proposals</u>](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#voting-rfc-proposals)).*

Reviewer: This feels long (and maybe kind of boring?), and people can always go read the whole thing. You've already *said* most of this, and I don't think anyone cares about the specifics of the Stewards role for example.

I would honestly scrap the entire direct quote... UNLESS you want to use it to expand on any one direct aspect with its governance technical details/impact and/or describe what's specifically NEW for 2025 (since you mentioned it being formed in 2022). It's not the content of the governance role that's the story here, it's the intention behind it:

For example, since this is a post written by *you*, what about a framing to specifically describe the new role the Steward has in all this (as part of Astro's success story), to achieve:
- an actual story in all this, about how governance evolves to fit the current state of the project
- "hey all you governance nerds, this one's for you!" (you love this Fred, this is your moment! Lean into it!)
- CELEBRATING that OMG, there are now SO MANY PEOPLE who can step into membership here
- (subconsciously) Here's why you might see me less and Matt K more (e.g. at conferences etc.), I'm now finally free to focus on doing the boring (but essential!) admin/funding/governance behind the scenes stuff

Something that could take a form like:

> The TSC was formed to be ultimately responsible for the long-term health and growth of the Astro codebase. For three years, its members have been responsible for overseeing the RFC (Request for Comments) process: considering items on [Astro's public roadmap](https://github.com/withastro/roadmap), weighing all technical considerations, and ultimately choosing which features to develop next through consensus. This committee has a huge impact on the direction of Astro!
>
> In 2025, we revisited this model for the first time since its creation, and [added a new clause](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#technical-steering-committee-tsc) to better align with our expanded team:
> 
> *The Steward is not eligible for the TSC, but may participate in TSC discussions and has a role in guiding the conversation to consensus (see [<u>Voting: RFC Proposals</u>](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#voting-rfc-proposals)).*
>
> When the TSC was originally formed, Astro's Project Steward (Hi, that's me! 👋) was one of the very few team members to satisfy the criteria of having demonstrated "a significant contribution history of code to the Astro project on GitHub." But, that's no longer the case! This new structure allows some of our most active (trusted, invested, capable?) core contributors to focus on the codebase, allowing the role of Steward to focus on its own, additional responsibilities that are crucial to the success of our open source project like administration and funding.
>
> I am so excited that the Astro open source project has reached the stage where we have more than enough capable people to fill all the roles we carefully crafted in the beginning: roles we knew were essential to sustaining a project... (TIE THIS ALL INTO HOW SUCCESSFUL ASTRO IS) -- maybe the newest download numbers we've been seeing?

Something like above feels like *a reason to write this post*, and a reason to write it *now*.

Author: Will definitely bring some of this into the post!

Author: Did another pass, would love your thoughts. I really like re-focusing the middle part of the section on the updates this year, that's so much better than the blockquote.

Overall though I'm intentionally not doing too much storytelling here and keeping the framing that this is primarily an announcement post vs. a think piece. There is ABSOLUTELY a great think piece to be written about how our governance evolved over the last 4 years and what we learned, but IMO that would stand better on its own.

Reviewer: Love the updated version! I think it really adds a nice focus here!

Open source governance sometimes gets a bad reputation for being slow or bureaucratic. But in our case, it's been the opposite. Having clear roles and responsibilities has helped us move *faster* — especially on a distributed, async team like ours

Our TSC model was inspired by [ESLint](https://eslint.org/) with guidance from [Nicholas Zakas](https://humanwhocodes.com/blog/), who is a brilliant open source maintainer and project steward that we have been lucky to learn from.

## Get Involved

Each TSC member — Ema, Matt, Florian, and Matthew — has already made major contributions to Astro and played a significant role in growing the framework into what it is today. We are lucky and grateful for their continued leadership of the project.

> **The Astro project is proudly open source.** If you have ever considered contributing to an open source project, now is a great time to start. Learn more about our governance model and join our community [on Discord](https://astro.build/chat) to get involved.

Reviewer: ```suggestion
But, you don't have to be a core maintainer, or have a fancy role to contribute to Astro! Everyone started somewhere!

**The Astro project is proudly open source.** If you have ever considered contributing to open source, now is a great time to start and Astro is a great place to get started. You can:

- Look through our [open issues](https://github.com/withastro/astro/issues) and help with making minimal reproductions, or working on fixes.
- Get involved with [open RFC discussions](https://github.com/withastro/roadmap/discussions) (or, submit your own!) to help plan future features.
- Contribute to [Astro's documentation and translation](https://github.com/withastro/docs/) so that people can be successful using Astro.
- Join our community [on Discord](https://astro.build/chat) to help answer support questions.

And don't forget to grab your own [Astro contributor badge](https://astro.badg.es/contributors/) — if you've made a contribution to any of Astro's repos, your name and achievements will be there.

Learn more about [Astro's open source governance model](https://github.com/withastro/.github/blob/main/GOVERNANCE.md) and join our community [on Discord](https://astro.build/chat) to get involved.
```
This feels like a place where it makes sense to more specifically guide people as to *how* they can contribute. (If saying "get started" they probably need a bit more direction than that!) But also, the specifics are a nice reminder, and calling out things that seem possible and achievable are encouraging, even to our current community.

I would also take this opportunity to mention Astro badges! It's a very cool thing, and it needs better discoverability. This also nicely pivots and wraps up a post that started out as "Here are your overlords" with "and here's how you can join the club."

---
title: '2025 Technical Steering Committee'
description: 'Meet Astro’s 2025 Technical Steering Committee and learn how our open governance model supports the framework’s growth, performance, and community-led development.'
publishDate: '2025-04-18'
authors:
  - fred
coverImage: '/src/content/blog/_images/astro-tsc-2025/blog-hero.jpg'
socialImage: '/src/content/blog/_images/astro-tsc-2025/og-image.jpg'
lang: 'en'
---

import Mention from "/src/components/Mention.astro"

> The Astro project has a new Technical Steering Committee (TSC) for 2025! I am excited to announce <Mention name="ema" /> <Mention name="florian" /> and <Mention name="matt" /> are now part of the Astro TSC, joining Matthew Phillips who will remain on the TSC. 

Reviewer: Just pointing out that Matthew is the only one here not `<Mentioned>`, in case that wasn't intentional!

Author: whoops! Thanks!

In addition, I am also proud to recognize <Mention name="matt" /> as the official Astro framework team lead going forward, joining <Mention name="sarah" /> who will continue as the project's official docs team lead.

> This announcement is part of our ongoing commitment to open governance and bringing you the best framework for building content-driven websites. If you're curious to learn more, you can read our entire set of public governance docs [on GitHub](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#get-involved), see our [full list of maintainers](https://github.com/withastro/.github/blob/main/MEMBERS.md), and chat with us [on Discord](https://astro.build/chat).

Reviewer: ```suggestion
This announcement is part of our ongoing commitment to open governance and bringing you the best framework for building content-driven websites. If you're curious to learn more, you can read our entire set of public [governance docs on GitHub](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#get-involved), see our [full list of maintainers](https://github.com/withastro/.github/blob/main/MEMBERS.md), and chat with us [on Discord](https://astro.build/chat).
```
This might have gotten lost in the rewrite, but I still suggest making the governance docs part of the linked text! (It will stand out more, and make it clear it's not just the repo for people who are skimming)

## What is the Astro TSC?

> We created the Technical Steering Committee (TSC) in 2022 [(PR #3490)](https://github.com/withastro/astro/pull/3490) to support Astro’s rapid growth in the early days of the project. Astro usage was soaring — npm downloads were doubling monthly — and our core team of maintainers had expanded to include new members with new skillsets (docs, design, community-building) outside of just coding. Astro needed a smaller, focused group to make core technical decisions more effectively.

Reviewer 2: "skill set" is two separate words
```suggestion
We created the Technical Steering Committee (TSC) in 2022 [(PR #3490)](https://github.com/withastro/astro/pull/3490) to support Astro’s rapid growth in the early days of the project. Astro usage was soaring — npm downloads were doubling monthly — and our core team of maintainers had expanded to include new members with new skill sets (docs, design, community-building) outside of just coding. Astro needed a smaller, focused group to make core technical decisions more effectively.
```

> > *A TSC member guides the direction of the project and ensures a healthy future for the Astro codebase. TSC members are ultimately responsible for technical decision making when it comes to any changes to the Astro codebase.*

Reviewer 2: hyphen
```suggestion
> *A TSC member guides the direction of the project and ensures a healthy future for the Astro codebase. TSC members are ultimately responsible for technical decision-making when it comes to any changes to the Astro codebase.*
```

When you talk about governance in open source, it's easy to assume that all of this is about slow, boring bureaucracy. But in our case, it's had the exact opposite effect. Having clear roles and responsibilities has helped us move *faster* with focus — especially on a distributed, async team like Astro. Our TSC model was inspired by [ESLint](https://eslint.org/) with early guidance from [Nicholas Zakas](https://humanwhocodes.com/blog/), who is a brilliant open source maintainer and project steward that we have been lucky to learn from.

## Get Involved

**The Astro project is proudly open source.** If you have ever considered contributing to an open source project, now is a great time to start. Learn more about our governance model [on GitHub](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#get-involved), see our [full list of maintainers](https://github.com/withastro/.github/blob/main/MEMBERS.md), and join us [on Discord](https://astro.build/chat) to say hi and learn more about how you can get involved.

> Join 5,000+ other Astro contibutors and grab your own [Astro contributor badge](https://astro.badg.es/contributors/) — if you've helped out in any of Astro's repos on GitHub, your name and achievements will be listed there.

Reviewer 2: typo
```suggestion
Join 5,000+ other Astro contributors and grab your own [Astro contributor badge](https://astro.badg.es/contributors/) — if you've helped out in any of Astro's repos on GitHub, your name and achievements will be listed there.
```
